### ISSUE TITLE
Please make a way to make a community on juxt.

### ISSUE THREAD

**Issue by sebexyt155** [PretendoNetwork/.github#87]
### Checked Existing

- [x] I have checked the repository for duplicate issues.

### What feature do you want to see added?

I just would want a way to make a option to make your own community. I mean for example: log in on a computer to juxt and make a community from there

### Why do you want to have this feature?

So. Do you want to juxt end like original miiverse? I mean that none communities were made by the community so people went on reddit, twitter etc.
I know that juxt was supposed to be just like miiverse but even you made some changes to the interface. So one change will be a big deal. But for community. And in good meaning

**Comment by jonbarrow** [comment]
I do not believe we have any current plans to allow user-made communities. Will CC @CaramelKat on that however

That being said, this is the wrong repo for such a request. I'm going to close this under the assumption that we don't have plans for this feature, unless corrected by Jemma
